apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -488,12 +488,13 @@ function wpcom_display_global_styles_launch_bar( $bar_controls ) {
 				<div class="launch-bar-global-styles-message">
 					<?php
 					$message = sprintf(
-						/* translators: %s - documentation URL. */
+						/* translators: %1$s - documentation URL, %2$s - the name of the required plan */
 						__(
-							'Your site includes <a href="%s" target="_blank">premium styles</a> that are only visible to visitors after upgrading to the Premium plan or higher.',
+							'Your site includes <a href="%1$s" target="_blank">premium styles</a> that are only visible to visitors after upgrading to the %2$s plan or higher.',
```

Comment:
@fushar do you think we should change `premium styles` to `style variations` to be more aligned with the PR https://github.com/Automattic/wp-calypso/pull/85513?

Reply:
Hmm, not sure. This launchbar will also show if the user applies a global styles (like changing fonts), not limited to style variations only.

Reply:
If anything, I think "custom styles" is more correct term, but I think we can tackle it later 😄 we should take a note!